frontend/src/lib-customizations/types.d.ts

```diff
@@ -12,6 +12,12 @@ export interface CitizenCustomizations {
     src: string
     alt: string
   }
+  mapSearchAreaRect: {
+    minLongitude: number
+    maxLongitude: number
+    minLatitude: number
+    maxLatitude: number
+  }
```

Comment:
I'd rather put these into `MapConfig` as that is just our own config type for everything related to the map.